app/models/manageiq/providers/openstack/infra_manager/host.rb

```diff
@@ -412,11 +412,119 @@ def provide(taskid = nil)
       begin
         MiqEvent.raise_evm_job_event(self, :type => "provide", :suffix => "complete")
       rescue => err
-        $log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
+        _log.warn("Error raising complete provide event for #{log_target}: #{err.message}")
       end
     end
 
     task.update_status("Finished", task_status, "Provide Complete with #{workflow_state}") if task
-    $log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+    _log.info("Provide #{log_target}...Complete - Timings: #{t.inspect}")
+  end
+
+  def supports_start?
```

Comment:
@durandom do we need also class definition for the supports? I think so

Also it should be probably host_start and host_stop from the feature list https://github.com/Ladas/manageiq/blob/c262eb6c5c3dec1c05a2018d6972116233ebdb6c/db/fixtures/miq_product_features.yml#L1041-L1041

and you need to list it also here I think https://github.com/Ladas/manageiq/blob/9bab4514bc56450a031bdd8da1202595132ea0de/app/models/mixins/supports_feature_mixin.rb#L60-L60